src/mlpack/methods/sparse_autoencoder/maximal_inputs_impl.hpp

```diff
@@ -32,10 +32,10 @@ inline void NormalizeColByMax(const arma::mat &input,
   output.set_size(input.n_rows, input.n_cols);
   for (arma::uword i = 0; i != input.n_cols; ++i)
   {
-    const double max = arma::max(arma::abs(input.col(i)));
-    if (max != 0.0)
+    const double max_ = max(arma::abs(input.col(i)));
+    if (max_ != 0.0)
     {
-      output.col(i) = input.col(i) / max;
+      output.col(i) = input.col(i) / max_;
```

Comment:
```suggestion
      output.col(i) = input.col(i) / colMax;
```
Goes with the suggestion above.